app/TestTCP/testcongestion.cpp

```diff
@@ -436,15 +440,13 @@ TEST_F(TestEnv_Congestion2, TestCongestion2)
 		Time start_time = TimeUtil::makeTime(1,TimeUtil::SEC);
 		start_time += TimeUtil::makeTime(0,TimeUtil::SEC);
 
-		accept_env["RANDOM_SEED"] = "104729";
```

Comment:
change it to `RANDOM_SEED = rand()`, and set both accept_env and connect_env with the seed value.